src/promptflow/promptflow/_sdk/data/docker_csharp/runit/promptflow-serve/run

```diff
@@ -2,4 +2,4 @@
 
 echo "start promptflow serving"
 cd /flow
-dotnet Promptflow.DotnetService.dll --port "8080" --yaml_path "flow.dag.yaml" --assembly_folder "." --connection_provider_url "" --log_path "" --serving
\ No newline at end of file
+dotnet Promptflow.dll --port "8080" --yaml_path "flow.dag.yaml" --assembly_folder "." --connection_provider_url "" --log_path "" --serving
```

Comment:
please confirm with Qian if this correct. and merge when there code is merged to main.